webui/react/src/components/ConfigPolicies.tsx

```diff
@@ -0,0 +1,146 @@
+import Alert from 'hew/Alert';
+import Button from 'hew/Button';
+import CodeEditor from 'hew/CodeEditor';
+import Column from 'hew/Column';
+import Form, { hasErrors } from 'hew/Form';
+import Row from 'hew/Row';
+import Spinner from 'hew/Spinner';
+import { useToast } from 'hew/Toast';
+import useConfirm from 'hew/useConfirm';
+import { Loadable, NotLoaded } from 'hew/utils/loadable';
+import yaml from 'js-yaml';
+import { useState } from 'react';
+
+import { useAsync } from 'hooks/useAsync';
+import usePermissions from 'hooks/usePermissions';
+import { getWorkspaceConfigPolicies, updateWorkspaceConfigPolicies } from 'services/api';
+import handleError from 'utils/error';
+
+interface Props {
+  workspaceId?: number;
+}
+
+type FormInputs = {
+  task: string;
+};
+
+const ConfigPolicies: React.FC<Props> = ({ workspaceId }: Props) => {
+  const confirm = useConfirm();
+  const { openToast } = useToast();
+  const { canModifyWorkspaceConfigPolicies, loading: rbacLoading } = usePermissions();
+  const [form] = Form.useForm<FormInputs>();
+
+  const [disabled, setDisabled] = useState(true);
+
+  const APPLY_MESSAGE = "You're about to apply these config policies to this workspace.";
+  const VIEW_MESSAGE = 'An admin applied these config policies to this workspace.';
+
+  const updatePolicies = async () => {
+    if (workspaceId) {
+      try {
+        await updateWorkspaceConfigPolicies({
+          configPolicies: form.getFieldValue('task'),
+          workloadType: 'NTSC',
```

Comment:
Just to confirm, the `workloadType` should always be NTSC?

Reply:
this PR is only for NTSC. The intent is to add UI for experiment type later.